src/ApiProblem.php

```diff
@@ -250,14 +250,24 @@ protected function getDetail()
     }
 
     /**
```

Comment:
Throwing my 2 cents in here, I don't like the term 'raw' in my applications.  I'm in favor of finding a better function name.